lib/resources/etc_libuser.rb

```diff
@@ -0,0 +1,108 @@
+require 'utils/simpleconfig'
+require 'utils/object_traversal'
+
+module Inspec::Resources
+  class EtcLibUser < Inspec.resource(1)
+    name 'etc_libuser'
+    desc 'Use the etc_libuser InSpec audit resource to check the configuration of /etc/libuser.conf.
+          Due to limitations of the RSpec engine and for consistency, please use the method described here
+          (https://github.com/chef/inspec/issues/1543) for accessing the hierarchical values in
+          /etc/libuser.conf'
+    example "
+          ---start sample /etc/libuser.conf---
+          # See libuser.conf(5) for more information.
+          ...
+          [import]
+          # Data from these files is used when libuser.conf does not define a value.
+          # The mapping is documented in the man page.
+          login_defs = /etc/login.defs
+          default_useradd = /etc/default/useradd
+          ...
+          ---end sample /etc/libuser.conf---
+
+          describe etc_libuser do
+            its(['import','login_defs']){should eq ['/etc/login.defs']}
```

Comment:
Please provide spacing that is consistent with the other documentation and examples. I'd also like to request that you change this example to use the `cmp` matcher so users don't have to worry about array syntax unless necessary:

```ruby
its(['import', 'login_defs']) { should cmp '/etc/login.defs' }
```